src/Standards/Generic/Docs/Commenting/DocCommentStandard.xml

```diff
@@ -0,0 +1,268 @@
+    There must be no additional blank lines before the closing DocBlock tag.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: no additional blank lines before the closing DocBlock tag.">
+        <![CDATA[
+/**
+ * Short description.
+<em></em> */
+        ]]>
+        </code>
+        <code title="Invalid: additional blank lines before the closing DocBlock tag.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *</em>
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The DocBlock must have a short description, and it must be on the first line.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: DocBlock with a short description on the first line.">
+        <![CDATA[
+/**
+ * <em>Short description.</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: DocBlock without a short description or short description not on the first line.">
+        <![CDATA[
+/**
+ * <em></em>@return int
+ */
+
+/**
+<em> *</em>
+ * Short description.
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The short and long descriptions must start with a capital letter.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Short and long descriptions start with a capital letter.">
+        <![CDATA[
+/**
+ * <em>S</em>hort description.
+ *
+ * <em>L</em>ong description.
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Short or long descriptions don't start with a capital letter.">
+        <![CDATA[
+/**
+ * <em>s</em>hort description.
+ *
+ * <em>l</em>ong description.
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There must be exactly one blank line separating the short description, the long description and tag groups.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: One blank line separating the short description, the long description and tag groups.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *<em>
+ * Long description.
+<em> *</em>
+ * @param int $foo
+ */
+        ]]>
+        </code>
+        <code title="Invalid: More than one or no blank line separating the short description, the long description and tag groups.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *
+ *
+</em>
+ * Long description.
+ * @param int $foo
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Parameter tags must be grouped together.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Parameter tags grouped together.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+<em> * @param int $foo
+ * @param string $bar</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Parameter tags not grouped together.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+ * @param int $foo
+<em> *</em>
+ * @param string $bar
```

Comment:
My understanding is that you are suggesting to change this example to:

```php
/**
 * Short description.
 *
 * @param int $foo
 * @since 1.2.0
 * @param string $bar
 */
```

If that is the case, I don't think we should make this change as this triggers a different error than the error described in this `<standard>`. The current example triggers `Generic.Commenting.DocComment.ParamGroup` while the example you are suggesting triggers `Generic.Commenting.DocComment.NonParamGroup` and `Generic.Commenting.DocComment.TagsNotGrouped`.

We could change this example to:

```php
/**
 * Short description.
 *
 * @param int $foo
 * 
 * @since 1.2.0
 * 
 * @param string $bar
 */
```

But I'm not sure if it would make the example more clear, and besides `Generic.Commenting.DocComment.ParamGroup`, this example also triggers `Generic.Commenting.DocComment.TagsNotGrouped`. So I'm inclined to leave the example as is. What do you think?